src/style.css

```
@@ -0,0 +1,36 @@
+* {
+  margin: 0;
+}
+
+.box {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  width: 96px;
+  height: 96px;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--1 :first-child,
+.stars--2 > :nth-child(-n + 2),
+.stars--3 > :nth-child(-n + 3),
+.stars--4 > :not(:last-child),
```

Let's be consistent and use nth-child here too. Because if there will be 10 stars, for example, you would have to rewrite it.
```suggestion
.stars--4 > :nth-child(-n + 4),
```